docs/iam_roles.md

```diff
@@ -139,3 +141,45 @@ You can have an additional policy for each kops role (node, master, bastion). Fo
         }
       ]
 ```
+
+## Use existing AWS Instance Profiles
+
+Rather than having Kops create and manage IAM roles and instance profiles, it is possible to use an existing instance profile. This is useful in organizations where security policies prevent tools from creating their own IAM roles and policies.
+**Be sure to compare externally-managed policies against Kops-managed policies anytime Kops is upgraded as this can cause very unusual behavior if policies are not as expected.**
+
+*NOTE: Currently Kops only supports using existing instance profiles for every instance group in the cluster, not a mix of existing and managed instance profiles.
+This is due to the lifecycle overrides being used to prevent creation of the IAM-related resources.*
+
+To do this, get a list of instance group names for the cluster:
+
+```
+kops get ig --name ${CLUSTER_NAME}
+```
+
+And update every instance group's spec with the desired instance profile ARNs:
+
+```
+kops edit ig --name ${CLUSTER_NAME} ${INSTANCE_GROUP_NAME}
+```
+
+Adding the following `iam` section to the spec:
+
+```yaml
+spec:
+  iam:
+    profile: arn:aws:iam::1234567890108:instance-profile/kops-custom-node-role
+```
+
+Now run a cluster update to create the new launch configuration, using [lifecycle overrides](./cli/kops_update_cluster.md#options) to prevent IAM-related resources from being created:
+
+```
+kops update cluster ${CLUSTER_NAME} --yes --lifecycle-overrides "IAMRole=Ignore,IAMRolePolicy=Ignore,IAMInstanceProfileRole=Ignore"
```

Comment:
When testing having quotes around --lifecycle-overrides "IAMRole=Ignore,IAMRolePolicy=Ignore,IAMInstanceProfileRole=Ignore" makes it fail. Removing the quotes fixed this.